editor/store/effects.js

```diff
@@ -309,20 +326,11 @@ export default {
 			return;
```

Comment:
The line above this: We check `isEditedPostSaveable`. Shouldn't this be `isEditedPostAutosaveable` ? Do we allow autosave to occur while autosave is already in progress? What happens then to the value of state's `isAutosaving` when the first autosave returns (i.e. triggers `RESET_AUTOSAVE`) but the second hasn't?

Reply:
With the changes in 7b4bb19, should the `AUTOSAVE` effect be checking `isEditedPostAutosaveable` instead of `isEditedPostSaveable` (which now includes `isEditedPostSaveable` anyways)?

Reply:
Maybe worth testing the behavior of <kbd>Cmd+S</kbd>, which [fires itself as `autosave`](https://github.com/WordPress/gutenberg/blob/master/editor/components/editor-global-keyboard-shortcuts/index.js#L128), where we rely on the effect to stop itself from an unnecessary save.

**Edit:** Particularly, what happens when I press <kbd>Cmd+S</kbd> in quick succession (second press before the first autosave completes).